modules/cnf-provisioning-real-time-and-low-latency-workloads.adoc

```diff
@@ -0,0 +1,242 @@
+// CNF-489 Real time and low latency workload provisioning
+// Module included in the following assemblies:
+//
+// *cnf-performance-addon-operator-for-low-latency-nodes.adoc
+
+[id="cnf-provisioning-real-time-and-low-latency-workloads_{context}"]
+= Provisioning real-time and low latency workloads
+
+Many industries and organizations need extremely high performance computing and may require low and predictable latency,
+especially in the financial and telecommunications industries.
+For these industries, with their unique requirements, OpenShift Container Platform (OCP) provides a
+Performance Addon Operator to implement automatic tuning in order to achieve low latency performance and
+consistent response time for OpenShift applications.
+The cluster administrator uses this performance profile configuration that makes it easier to make these changes
+in a more reliable way.
+The administrator can specify whether to update the kernel to kernel-rt (real-time), the CPUs that will be reserved
+for housekeeping, and the CPUs are used for running the workloads.
+
+== Known limitations for real-time
```

Comment:
AIUI, we are not currently attempting to support control plane nodes running kernel-rt. We should be explicit about that here, if true.

See https://bugzilla.redhat.com/show_bug.cgi?id=1887007#c25 - a user attempting to use a `kernelType: realtime` MachineConfig at install time